wzdx/tools/wzdx_translator.py

```diff
@@ -196,3 +195,58 @@ def string_to_number(field):
             return float(field)
         except ValueError:
             return None
+
+
+# function to parse direction from street name
+def parse_direction_from_street_name(street):
+    if not street or type(street) != str:
+        return None
+    street_char = street[-1]
+    street_chars = street[-2:]
+    if street_char == 'N' or street_chars == 'NB':
+        direction = 'northbound'
+    elif street_char == 'S' or street_chars == 'SB':
+        direction = 'southbound'
+    elif street_char == 'W' or street_chars == 'WB':
+        direction = 'westbound'
+    elif street_char == 'E' or street_chars == 'EB':
+        direction = 'eastbound'
+    else:
+        direction = None
+
+    return direction
+
+
+def get_wzdx_schema(schema_file_name):
```

Comment:
The only place I see this new method referenced is in the `wzdx\tools\__init__.py`. Is this used somewhere outside this code base?

Reply:
This is not used, I created it during my quest to make the json schema files accessible in the package. My solution was to make them python files so it would be forced to keep them, and this function is useless now :(